dice_ml/explainer_interfaces/dice_genetic.py

```diff
@@ -306,6 +309,20 @@ def predict_fn(self, input_instance):
         input_instance = self.label_decode(input_instance)
         return self.model.get_output(input_instance, model_score=False)
 
+    def _predict_fn_custom(self, input_instance, desired_class):
```

Comment:
can you write some comments for the logic here? saying that why we should prefer the output to be `desired_class`, and we actually change it. (and why it is needed--the issue with sklearn's predict)